README.md

```diff
@@ -29,3 +29,12 @@ gp_relsizes_stats configuration parameters:
 | `gp_relsizes_stats.database_naptime` | int     | 0        | Using `gp_relsizes_stats.database_naptime` you can set naptime between collecting stats for each databases. Value set time in milliseconds. Default is equal to 0 milliseconds.|
 | `gp_relsizes_stats.file_naptime`     | int     | 1        | Using `gp_relsizes_stats.file_naptime` you can set naptime between each file stats calculating. Value set time in milliseconds. Default is equal to 1 millisecond.|
 
+### About collected data and tables
+All tables store actual info about database, but it will be truncated after each collecting operation restart, so for some historical data, check relsizes_stats_schema.table_sizes_history table for some non-actual info (actual for previous restart).
```

Comment:
For simplicity I'd remove this line completely. And in the table below I'd move table_sizes to the end of table and table_sizes_history right after it. On the third column it should be enough to describe what this table. I think it's okay to omit column descriptions here. For example:
| relsizes_stats_schema.table_sizes | (nspname, relname, relsize) | Current size of each table |
| relsizes_stats_schema.table_sizes_history | (nspname, relname, relsize) | Same as table_sizes but from a previous run. Can be used to see how table size changed since last script run or to accumulate historical data into another, longer living, table |